systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1133,6 +1134,41 @@ void testNodePort() throws Exception {
         waitForClusterAvailability(NAMESPACE);
     }
 
+    @Test
+    void testOverrideNodePortConfiguration() throws Exception {
+        LOGGER.info("Setting broker ID and NodePort");
```

Comment:
It will be useful to have more info in the logs like for which broker we override notePort value:
`LOGGER.info("Setting nodePort to {} for broker {}", nodePortValue, brokerId);`
Same for other logs in the test.

Reply:
Sure, that's a good idea. Done.